pkgs/development/libraries/vte/default.nix

```diff
@@ -93,7 +93,9 @@ stdenv.mkDerivation rec {
   ];
 
   # error: argument unused during compilation: '-pie' [-Werror,-Wunused-command-line-argument]
-  NIX_CFLAGS_COMPILE = lib.optionalString stdenv.hostPlatform.isMusl "-Wno-unused-command-line-argument";
+  NIX_CFLAGS_COMPILE = lib.optionalString stdenv.hostPlatform.isMusl "-Wno-unused-command-line-argument"
+    # https://github.com/NixOS/nixpkgs/issues/201254
+    + lib.optionalString (stdenv.isLinux && stdenv.isAarch64 && stdenv.cc.isGNU) "-lgcc";
```

Comment:
Should not there be a space or is musl orthogonal to GNU?

Reply:
The latter. And the flag also seems to be needed when building with musl + aarch64 + gcc, according to https://github.com/NixOS/nixpkgs/pull/193930#issuecomment-1264706849.